bin/save_cache

```diff
@@ -48,7 +48,17 @@ if ! aws s3api head-object --bucket "$S3_BUCKET_NAME" --key "$CACHE_KEY" > /dev/
 	echo "No existing cache entry for $CACHE_KEY – storing in cache"
 
 	echo "	Compressing"
-	tar -czf "$CACHE_KEY" "$CACHE_FILE"
+	TAR_OPTIONS=${4-foo}
```

Comment:
Why `-foo` here?

Reply:
Oops, that was left over from some things I was trying out 🙂

Reply:
Fixed with https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/43/commits/8a6377fc20ff835c4b2d4f9272d761548c110abc